commcare_connect/templates/tailwind/base_table.html

```diff
@@ -1,113 +1,165 @@
-                    <a href="{% querystring table.prefixed_page_field=table.page.previous_page_number %}" class="px-3 py-2 border border-gray-300 rounded-md text-sm font-medium text-gray-700 hover:bg-gray-50">
-                        <span aria-hidden="true">&laquo;</span>
-                        {% trans 'previous' %}
-                    </a>
-                </li>
-                {% endblock pagination.previous %}
-            {% endif %}
-            {% if table.page.has_previous or table.page.has_next %}
-            {% block pagination.range %}
-            {% for p in table.page|table_page_range:table.paginator %}
-                <li>
-                    <a class="px-3 py-2 border border-gray-300 rounded-md text-sm font-medium text-gray-700 hover:bg-gray-50 {% if table.page.number == p %}bg-gray-200{% endif %}" {% if p != '...' %}href="{% querystring table.prefixed_page_field=p %}"{% endif %}>
-                        {{ p }}
-                    </a>
-                </li>
+      {% block table.tbody %}
+      <tbody>
+            <div style="overflow:hidden; border-radius:8px;">
+          {% for row in table.paginated_rows %}
+          {% block table.tbody.row %}
+          <tr>
+            {% for column, cell in row.items %}
+            <td>
+              {% if column.localize == None %}
+                {{ cell }}
+              {% else %}
+                {% if column.localize %}
+                  {{ cell|localize }}
+                {% else %}
+                  {{ cell|unlocalize }}
+                {% endif %}
+              {% endif %}
+            </td>
             {% endfor %}
-            {% endblock pagination.range %}
-            {% endif %}
-            {% if table.page.has_next %}
-                {% block pagination.next %}
-                <li>
-                    <a href="{% querystring table.prefixed_page_field=table.page.next_page_number %}" class="px-3 py-2 border border-gray-300 rounded-md text-sm font-medium text-gray-700 hover:bg-gray-50">
-                        {% trans 'next' %}
-                        <span aria-hidden="true">&raquo;</span>
-                    </a>
-                </li>
-                {% endblock pagination.next %}
-            {% endif %}
-            </ul>
-        </nav>
-        {% endif %}
-    {% endblock pagination %}
+          </tr>
+          {% endblock table.tbody.row %}
+          {% empty %}
+          {% if table.empty_text %}
+          {% block table.tbody.empty_text %}
+          <tr>
+            <td colspan="{{ table.columns|length }}" class="text-center py-4">
+              {{ table.empty_text }}
+            </td>
+          </tr>
+          {% endblock table.tbody.empty_text %}
+          {% endif %}
+          {% endfor %}
+        </div>
+        </tbody>
+        {% endblock table.tbody %}
+
+    </table>
+  </div>
+  {% endblock table %}
 </div>
-{% endblock table-wrapper %}
\ No newline at end of file
+<style>
+    .scrollable-table-wrapper {
```

Comment:
We shouldn't be using raw CSS, right? We should use tailwind classes directly where it's needed on the base-table.

Reply:
additionally, if there needs to be this much css, it should probably be in a dedicated file, rather than at the bottom of the  table html